flytekit/clients/auth/keyring.py

```diff
@@ -71,9 +71,8 @@ def delete(for_endpoint: str):
         try:
             _keyring.delete_password(for_endpoint, KeyringStore._access_token_key)
```

Comment:
should we just do three separate try/catches?  to make sure we always clean up all three things?

Reply:
Makes sense. Fixed.